scenarios/prometheus.yml

```diff
@@ -0,0 +1,23 @@
+config:
```

Comment:
Maybe also add a post action to check the pod count i.e 2 similar to https://github.com/openshift-scale/kraken/blob/823cecad6e801d3ee86580f1bc480c7a8ba5d237/scenarios/post_action_openshift-kube-apiserver.yml#L19?

Reply:
Fixed it